### ISSUE TITLE
15 tests failed. locale:el:parse (450.23) locale:el:parse (450.39) locale:el:parse (450.47) locale:el:parse (450.55) locale:el:parse (450.87) locale:el:parse meridiem (451.5) locale:el:parse meridiem (451.9) locale:el:parse meridiem (451.13) locale:el:parse meridiem (451.17) locale:el:parse meridiem (451.21) locale:el:parse meridiem (451.25) locale:tzl:suffix (1869.2) locale:tzl:now from now (1870.1) duration:JSON.stringify duration (2186.1) format:full expanded format is returned from abbreviated formats (2218.1) 

### ISSUE THREAD

**Issue by MarkMaldaba** [moment/moment#2546]
### Client info
```
Date String   : Mon Aug 10 00:15:19 UTC+0100 2015
Locale String : 10 August 2015 00:15:19
Offset        : 0
User Agent    : Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1; .NET CLR 2.0.50727; .NET CLR 3.0.4506.2152; .NET CLR 3.5.30729; .NET CLR 1.1.4322; .NET4.0C)
```

====

### locale:el:parse (450.23) 

??????S should be month 3

```javascript
// Expected 2
// Actual   7
7 === 2
```

====

### locale:el:parse (450.39) 

????S should be month 5

```javascript
// Expected 4
// Actual   7
7 === 4
```

====

### locale:el:parse (450.47) 

??????S should be month 6

```javascript
// Expected 5
// Actual   7
7 === 5
```

====

### locale:el:parse (450.55) 

??????S should be month 7

```javascript
// Expected 6
// Actual   7
7 === 6
```

====

### locale:el:parse (450.87) 

????????S should be month 11

```javascript
// Expected 10
// Actual   7
7 === 10
```

====

### locale:el:parse meridiem (451.5) 

10 should be 22

```javascript
// Expected 22
// Actual   10
10 === 22
```

====

### locale:el:parse meridiem (451.9) 

10 should be 22

```javascript
// Expected 22
// Actual   10
10 === 22
```

====

### locale:el:parse meridiem (451.13) 

10 should be 22

```javascript
// Expected 22
// Actual   10
10 === 22
```

====

### locale:el:parse meridiem (451.17) 

10 should be 22

```javascript
// Expected 22
// Actual   10
10 === 22
```

====

### locale:el:parse meridiem (451.21) 

10 should be 22

```javascript
// Expected 22
// Actual   10
10 === 22
```

====

### locale:el:parse meridiem (451.25) 

10 should be 22

```javascript
// Expected 22
// Actual   10
10 === 22
```

====

### locale:tzl:suffix (1869.2) 

Died on test #2 undefined: Object doesn't support this property or method

```javascript
// Expected undefined
// Actual   null
null === undefined
```

====

### locale:tzl:now from now (1870.1) 

Died on test #1 undefined: Object doesn't support this property or method

```javascript
// Expected undefined
// Actual   null
null === undefined
```

====

### duration:JSON.stringify duration (2186.1) 

Died on test #1 undefined: 'JSON' is undefined

```javascript
// Expected undefined
// Actual   null
null === undefined
```

====

### format:full expanded format is returned from abbreviated formats (2218.1) 

Died on test #1 undefined: Object doesn't support this property or method

```javascript
// Expected undefined
// Actual   null
null === undefined
```

**Comment by MarkMaldaba** [comment]
I'm not sure if IE7 is officially supported, but with only 15 failing tests (most of which look like they come from the same cause) it is probably worth fixing these, if possible.

Note that the duration:JSON.stringify error looks like a regression of #867, and is probably an error in the tests.

**Comment by mj1856** [comment]
Well, I'm not sure we've been very official about it, but we do state [in the docs](http://momentjs.com/docs/#/use-it/)

> Currently the following browsers are used for the ci system: IE8, IE9 on Windows 7, stable Chrome on Windows XP, Safari 10.8 on Mac and stable Firefox on Linux.

Personally, I feel that IE8 is the minimum bar, and only because IE9 doesn't run on Windows XP.  Even that is becoming less important these days, given XP's end of life.

If the fix doesn't add much overhead, then sure - a PR would be welcome.  But if it's significant, and doesn't help any other environments, then it will probably be axed.

**Comment by MarkMaldaba** [comment]
Yeah - I saw that in the docs.  However, the conversation in issue #650 implies that IE7 was supported at the time (2013) and I couldn't find anywhere in the changelogs that indicated that IE7 support had been officially dropped.  Can you point me to the version where this happened?

As someone who provides software that is used by the NHS in the UK, where IE7 support is still (unfortunately) often still a requirement, I need a library that is not going to cause any breakage on IE7, otherwise I would have to look for an alternative solution.  An older version with fewer features would be acceptable, if there is a known-good release for IE7.

Note that 11 of the above failed tests were for the Greek language, 2 were for the constructed 'Talossan' language, and 1 is probably an invalid test (JSON object doesn't exist in IE7).  Therefore there is only one test that really matters here (from my perspective), which is the last one.
